# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoder.cpp
# Commit: 7d876e0ef72a
# Full Hash: 7d876e0ef72a5543c0c5ad207cde8b115f937036
# Author: alwu <alwu@mozilla.com>
# Date: 2022-10-18 09:48:31
# Regressor Bug: 1262276
# File Overlap Count: 1
# Description:
#   Bug 1262276 - part2 : support seamless video looping. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159126
# ==============================================================================

diff -r 70bc7e69d03a -r 7d876e0ef72a dom/media/MediaDecoder.cpp
--- a/dom/media/MediaDecoder.cpp	Mon Oct 17 18:28:16 2022 +0000
+++ b/dom/media/MediaDecoder.cpp	Mon Oct 17 18:28:17 2022 +0000
@@ -963,8 +963,7 @@
     double aPrevPos, double aCurPos) const {
   MOZ_ASSERT(NS_IsMainThread());
   // If current position is earlier than previous position and we didn't do
-  // seek, that means we looped back to the start position, which currently
-  // happens on audio only.
+  // seek, that means we looped back to the start position.
   const bool notSeeking = !mSeekRequest.Exists();
   if (mLooping && notSeeking && aCurPos < aPrevPos) {
     return PositionUpdate::eSeamlessLoopingSeeking;